language-design.md

```diff
@@ -18,6 +18,53 @@ For each quote, describe what it was about the quote that led you pick it.
+
+It is interesting, however, to consider Java as a counter-example. It was not until 2006 that Sun released most of its JVM as free and open-source. The development of Java as a programming language did not really follow Steele’s ‘bazaar-style’, yet was still widely used. This may have partly been because it was modeled on the syntax of C/C++, but intended to simplify certain things like memory management. In that sense, there were less risks involved in the creation of Java as opposed to earlier programming languages. 
+
+
+
+3. “Programming languages are tools, designed by people for a specific purpose. What this study showed is that the design of this particular tool, Perl, is so ridiculously opaque that, from the perspective of a novice programmer, a string of characters bashed out by a monkey at a keyboard would literally make an equal amount of sense… "We have observed that novices learning to program at the university or younger levels can have signiﬁcant difﬁculty learning the syntax of general purpose programming languages, which may initially seem arbitrary," the authors [of the study wrote].”
+
+We found this article, and in particular, this quote rather hard to believe so we decided to further investigate the validity of this claim in the additional optional reading.  It was rather surprising to see that the subjects performed better on coding tests using Randomo than Perl after being shown a code sample in their respective language (with an accompanying explanation of what the code does). Just looking at the code samples, it seems much easier to intuit how the Perl code functioned compared to Randomo. This may be because as programmers, certain syntax – like for loops - that look familiar to us are foreign to the test subjects who had no prior programming experience. Even though we had basically no exposure to Perl (Julien had seen a little bit), we were able to figure out how the Perl code worked. 
+
+Our main issue with Randomo is that it is not the “string of characters bashed out by a monkey” Pavlos rather dramatically portrays it to be. To quote the study: “Randomo: A programming language based largely on the syntactical structure of Quorum. With the exception of braces, the lexical rule for variable names, and a few operators (e.g., addition, subtraction, multiplication, division), many of the keywords and symbols were chosen randomly from the ASCII table.”
+
+This means that it by and large works syntactically as Quorum does – the language designed following “empirical studies on intuitiveness” -  with the biggest difference being the random keywords and symbols. To a complete beginner, constructs like for loops and symbols like “==” and “++” would have no inherent meaning and it’s almost understandable that random symbols such as those used in Randomo would at least be as easy to understand. 
+
+As a whole, we thought the main point of the article written by Pavlos was misleading. 
```

Comment:
Typical online journalism lol. But I think we (programmers) are biased because we're so accustomed to "abstruse syntax", while the testers had little programming experience. At the same time, I doubt that there can be purely "empirical studies on intuitiveness". It just seems too subjective.